src/Http/Url.php

```diff
@@ -77,11 +77,15 @@ public static function currentDir(): string
     /**
      * Tries to fix a broken url without protocol
      *
-     * @param string $url
+     * @param string|null $url
      * @return string
      */
     public static function fix(string $url = null): string
     {
+        if (empty($url) === true) {
+            return '';
+        }
+
```

Comment:
Possible breaking change?

Reply:
Why do you think so?

Reply:
Current

    Url::fix(''); // Returns: http://

Now

    Url::fix(''); // Returns: ''
    
If the variable is empty, there is nothing to fix.

Reply:
Since this usage does not make sense, we can call it a bug-fix.

Reply:
I wouldn't check for empty, but rather pass `$url ?? ''` to `preg_match`